schedule/sle-perf/hanaperf_installation.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: hanaperf_installation
+description: >
+  OS installation for HANA performance
+vars:
+  MULTIPATH: '1'
+  MULTIPATH_CONFIRM: 'yes'
+  SEPARATE_HOME: '1'
+  NUMDISKS: '7'
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - '{{scc_registration}}'
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_smalldisk_storageng
+  - '{{separate_home}}'
```

Comment:
As soon as you have `SEPARATE_HOME=1` always set (on 8th line of this yaml), I guess you do not need this conditional scheduling as this test suite will always have separate home.

So you can replace this {{separate_home}} with its content:
```
- installation/partitioning/no_separate_home
- '{{enlarge_swap}}'
```

Reply:
Actually, I need set the parameter, remove the hardcode in YAML file, it will be set by command line parameter for different machine and release.